locallib.php

```diff
@@ -1134,7 +1134,7 @@ function bigbluebuttonbn_generate_nonce() {
  * @return string
  */
 function bigbluebuttonbn_random_password($length = 8, $unique = "") {
-    $chars = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!@#$%^&*()_-=+;:,.?';
+    $chars = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!@#$%^*()_-=+;:,.?';
```

Comment:
This has drove me nuts, but it is a different PR.

I am opening an issu in the moodle tracker and will ask you to create anoter PR referencing it.